projboard/templates/base.html

```diff
@@ -49,6 +49,30 @@
             </li>
           </ul>
         </div>
+        <!-- SET UP ITS CSS LATER-->
+        {% if not user.is_authenticated %}
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link nav__link--btn" href="/signup"
+              >Open account</a
+            >
+          </li>
+        </ul>
+        {% else %}
+
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link nav__link--btn" href="/"
+              >Logout</a
+            >
+          </li>
+        </ul>
+        {% endif %}
```

Comment:
Great Work!
Seems like Yael also modifies the navbar in base.html for authenticated users and guests in PR #90 so maybe it's unnecessary here

Reply:
In my opinion it's ok, because we can resolve it in the merge conflict.

Reply:
Yes I can resolve conflicts after merging, no problem